app/views/course/forum/posts/edit.html.slim

```diff
@@ -0,0 +1,3 @@
+= page_header
+
+= render partial: 'form', locals: { url: course_forum_topic_post_path(current_course, @forum, @topic, @post) }
```

Comment:
Is it because when you do `render partial:`? What if you just did `render ‘partial’, url: xxx_path, …`